src/main/scala/ztis/testdata/MovieLensDataLoader.scala

```diff
@@ -0,0 +1,81 @@
+package ztis.testdata
+
+import java.io.File
+import java.net.URL
+
+import com.datastax.spark.connector._
+import com.typesafe.config.{Config, ConfigFactory}
+import com.typesafe.scalalogging.slf4j.StrictLogging
+import org.apache.spark.{SparkConf, SparkContext}
+import ztis.CassandraClient
+
+import scala.collection.JavaConverters._
+import scala.sys.process._
+
+object MovieLensDataLoader extends App with StrictLogging {
+
+  try {
+
+    val config = ConfigFactory.load("testdata")
+    bootstrapCassandra(config)
+    // downloadDataset(config)
```

Comment:
If this is commented out because we do not want to download the dataset everytime we execute the application maybe move the file name to the configuration and just create simple bash script that uses curl/wget to download the data. 

Also you could check if given file already exists - in that case do not download dataset again.